benches/benchmarks/check_texts.rs

```diff
@@ -48,7 +50,7 @@ async fn check_text_split(text: &str) -> Response {
 
     let resps = join_all(lines.map(|line| {
         async {
-            let req = Request::default().with_text(line.to_string());
+            let req = Request::default().with_text(Cow::Owned(line.to_string()));
```

Comment:
Same comment as above
```suggestion
            let req = Request::default().with_text(line.to_string());
```